File: Modul7/Soal1/PRAK601-2210817110008-Aditya Oktaviari.py
a, b= map(int, input().split())
nilai= list(map(int, input().split()))
matriks = []

for i in range(0, a*b, b):
    matriks.append(nilai[i : i + b])
print(" ")

for i in range(a):
    for j in range(b):
        print(matriks[i][j], end=' ')
    print()

File: Modul7/Soal1/PRAK601-2210817110008-Aditya Oktaviari.c
#include <stdio.h>

int main()
{
    int a, b, i, j, nilai;
    scanf("%d%d", &a, &b);
    int matriks[a][b];
    for (i=0; i<a; i++){
        for (j=0; j<b; j++){
            scanf("%d", &nilai);
            matriks[i][j]=nilai;
        }
    }
    printf("\n");

    for (i=0; i<a; i++){
        for (j=0; j<b; j++){
            printf("%d ", matriks[i][j]);
        }
        printf("\n");
    }
    return 0;

}
